### DISCUSSION TITLE
Deleting MAPs on Taglab

### DISCUSSION THREAD

**Discussion by CoralReefer135** [cnr-isti-vclab/TagLab#145] [Q&A]

**Comment by maxcorsini** [comment]
Dear CoralReefer135,  

you can delete a map by opening the Maps Editor. The Maps Editor show information on the orthoimages added to the project. 
The Edit and Delete buttons are on the right, if you do not see them use the horizontal scrollbar to go to the ends of the information box.

Hope it helps.

Best

**Reply by esme3093** [reply]
Hello, I'm having trouble with the Maps Editor tool. I want to delete some repetitive maps I added to a project, but the tool does not work anymore and makes Taglab crash completely. Was there a limit of maps that can be added in a single project? Any suggestions?
